src/DataObjects/WorkflowInstance.php

```diff
@@ -296,10 +296,9 @@ public function getTargetDiff()
         $diff->ignoreFields($this->config()->get('diff_ignore_fields'));
 
         $fields = ArrayList::create();
-        try {
+
+        if ($diff && !(is_null($diff->fromRecord) && is_null($diff->toRecord))) {
```

Comment:
Any particular reason to remove the try/catch block around the `$diff->ChangedFields();` section?

Reply:
I just thought that since the logs trace this stack just fine, we don't need the catch here (also no exception message in the catch). But I can add it back.

Reply:
Yeah please keep it, iirc that particular exception type is thrown in certain scenarios that are actually 'valid' from the point of view of the workflow - but if it's not caught, the whole workflow can get blocked from executing through.